onboarding-carts/jmeter/load.jmx

```diff
@@ -9,7 +9,7 @@
         <collectionProp name="Arguments.arguments">
           <elementProp name="SERVER_URL" elementType="Argument">
             <stringProp name="Argument.name">SERVER_URL</stringProp>
-            <stringProp name="Argument.value">carts.jx-staging.35.233.18.9.nip.io</stringProp>
+            <stringProp name="Argument.value">carts.sockshop-staging.public.demo.keptn.sh</stringProp>
```

Comment:
Same here - is the hostname really intended? We should refer to 127.0.0.1 here, afaik.

Reply:
yes - same as above